campuses-with-active-researchers/campuses-with-active-researchers.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Prints list of campuses with active researchers in the given date range, along with whether they're CC* or not.
+
+See https://opensciencegrid.atlassian.net/browse/SOFTWARE-4396 for
+the definition of 'active researcher'.
+
+"""
+
+from argparse import ArgumentParser
+import csv
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, A, Q
+import datetime
+import sys
+from typing import List, Optional, Set
+import urllib3
+import xml.etree.ElementTree as ET
+
+
+GRACC = "https://gracc.opensciencegrid.org/q"
+TOPOLOGY_RGSUMMARY = "https://topology.opensciencegrid.org/rgsummary/xml"
+
+
+def safe_elem_text(elem: Optional[ET.Element]) -> str:
+    """Return the stripped text of an element if available.  If not available, return the empty string"""
+    text = getattr(elem, "text", "")
+    return text.strip()
+
+
+def get_topology_resourcegroups() -> List[ET.Element]:
+    """Return the ElementTree Elements of the active, enabled ResourceGroups in the rgsummary XML"""
+    http = urllib3.PoolManager()
+    r = http.request(
+        "GET",
+        TOPOLOGY_RGSUMMARY,
+        fields={
+            "active": "on",
+            "active_value": "1",
+            "disable": "on",
+            "disable_value": "0",
```

Comment:
Same comment as https://github.com/path-cc/metric-tools/pull/2#discussion_r555921871:

> This took me a second to parse because our Topology interface is confusing. Could you add a comment clarifying the `active/disable: on` tuples are for enabling filtering based on the `active/disable` values?

Reply:
Added.